src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age !== undefined && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {!person.isMarried && <p>I am not married</p>}
+      {person.isMarried && (
+        person.sex === 'm'
+          ? (
+            `${person.partnerName} is my wife`
+          ) : (
+            `${person.partnerName} is my husband`
+          )
```

Comment:
According to previous comment and avoid linter errors this expression can be moved to variable and then just render here